x/migration/module/cmd/claim_morse_supplier.go

```diff
@@ -19,8 +20,8 @@ import (
 
 func ClaimSupplierCmd() *cobra.Command {
 	claimSupplierCmd := &cobra.Command{
-		Use:   "claim-supplier [morse_key_export_path] [path_to_supplier_stake_config] --from [shannon_dest_key_name]",
-		Args:  cobra.ExactArgs(2),
+		Use:   "claim-supplier [morse_operator_address] [morse_signer_key_export_path] [path_to_supplier_stake_config] --from [shannon_dest_key_name]",
```

Comment:
Per the table I put together in #1311, shouldn't `morse_operator_address` be more like ` morse_signer_address`?

So it's either the operator or the owner?

Reply:
See https://github.com/pokt-network/poktroll/pull/1328#discussion_r2093123633